docs/analyses/index.md

```diff
@@ -1,9 +1,9 @@
-# Analyses
+# Public analyses
 
 This section contains a selection of public analyses we want to share. This can include the methodology, replication code, and supplementary data or figures.
```

Comment:
Maybe good to be a bit clearer on what this page contains

> In this page, we publish technical reports on data projects at Our World in Data. These publications include methodology, replication code, and supplementary data or figures. The publications shared here may support live articles on our site, you can think of them as "technical appendices" of research articles.

(just drafted this quickly)